lib/middleware/index.js

```diff
@@ -132,7 +136,6 @@ export function all(options) {
   app.use(renderPageNotFound, renderErrorPage)
 
   return function (req, res, next) {
-    res.locals.serviceName = serviceName
```

Comment:
There's no middleware anymore

So we don't need `app.use()` around the `.all()` call

Reply:
am confused, sorry. The `serviceName` currently isn't being set in locals in `main`.

Reply:
The now-empty middleware function can entirely go

```patch
    // Page not found - this must come last
    app.use(renderPageNotFound, renderErrorPage)
- 
-   return function (req, res, next) {
-     res.locals.serviceName = serviceName
-     next()
-   }
```

Reply:
Ahh I get you now. Ok this is done.